web-console/knikubevirt/Create-vm/step-5-review/wizard-review.md

```diff
@@ -12,25 +12,48 @@ This encourages the user to check their settings quickly before creating the VM.
 
 ![user checks settings before creating vm](img/check-settings.png)
 
-When the user clicks “Create virtual machine” they will then be taken to the list view where they can see the VM being started and the state of the VM as "Starting".
+# States
+
+## creating State
+
+When the user clicks “Create virtual machine” they will then be will see the "creating state" for the virtual machine. In this step they would be removed from the navigation of the wizard and only able to get back in an error state where something needs to be addressed.
+
+![vm is being created](img/vm-creating.png)
+
+Alternatively, The user could navigate away from this page, In that case they'd see the VM in the list in a starting state. This will rely on the VM successfully being created without issue.
```

Comment:
Should we note here that VMs with issues will also be in the list and will be covered below?

Reply:
For the VMs with issue maybe add the aestrick following the status to indicate something is wrong

Reply:
@yfrimanm I don't think we'll need that b/c the status should show as an error or warning. I know we've done that for other instances (guest agent not installed) but that's more of subtle way to bring a users attention to it b/c it might have been intentional or they are already aware of it.